pcdsdevices/epics_motor.py

```diff
@@ -579,8 +577,9 @@ class IMS(PCDSMotorBase):
     velocity = Cpt(EpicsSignal, '.VELO', limits=True, kind='config')
     velocity_base = Cpt(EpicsSignal, '.VBAS', kind='omitted')
     velocity_max = Cpt(EpicsSignal, '.VMAX', kind='config')
+    acceleration = Cpt(EpicsSignal, '.ACCL', kind='config')
```

Comment:
This is already defined at https://github.com/bluesky/ophyd/blob/dd4b3e389a0202ecacce39fc3965d703c616b0d4/ophyd/epics_motor.py#L56, does something we do remove it?

Reply:
This is explicitly removing `auto_monitor=True` for some reason as well.

Reply:
```
$ ipython
Python 3.9.6 | packaged by conda-forge | (default, Jul 11 2021, 03:39:48)
Type 'copyright', 'credits' or 'license' for more information
IPython 7.26.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: import pcdsdevices

In [2]: pcdsdevices.__version__
Out[2]: '4.7.1'

In [3]: from pcdsdevices.epics_motor import IMS

In [4]: mot = IMS('XPP:USR:MMS:01', name='mot')
Using production server.

In [5]: mot.acceleration
Out[5]: EpicsSignal(read_pv='XPP:USR:MMS:01.ACCL', name='mot_acceleration', parent='mot', value=1.0, timestamp=1632762543.584904, tolerance=0.0001, auto_monitor=True, string=False, write_pv='XPP:USR:MMS:01.ACCL', limits=False, put_complete=False)
```